test/deprecated/functionality/README

```diff
@@ -0,0 +1,3 @@
+This directory is for tests of how to deprecate a symbol.  Its contents are
```

Comment:
Hmm. I'd maybe put this directory in `test/parsing/deprecated` instead or something like that.

Reply:
What about a new directory `test/deprecated-keyword` as a sibling of `test/deprecated`?

Reply:
That would be OK, modulo the issue of it making tab-completion more annoying. I recommend you ask 1 other person; if they like what you originally did better then maybe we stick with that.

Reply:
I asked David L and he agrees `test/deprecated/functionality` is bad and likes `test/deprecated-keyword` - autocomplete will stop at `test/deprecated`, which is the one people are more likely to want